chiron/tests/conftest.py

```diff
@@ -10,3 +10,21 @@ def remove_h5_file():
     yield
     if os.path.exists("test.h5"):
         os.remove("test.h5")
+
+
+@pytest.fixture
+def provide_testsystems_and_potentials():
```

Comment:
I think we might want to have different fixtures for LJ systems and the harmonic oscillators (single and array), and ideal gas (I think those were 3 we discussed as targets for testing)  Each have different methods in their openmm classes in terms of validation, so it will be hard to make fully generic tests.  I think separating them will also make it easier for debugging purposes.  A fixture for all test systems might be useful if we want to test against simulation generated data comparisons (i.e., Chiron results to openmm results).